test/functions/operatorOverloads/operatorMethods/genericsInstantiationBad.prediff

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 output=$2
 
-sed -e 's/:[0-9]*:/:N:/' $output > $output.tmp
+sed -e 's/:[0-9]*:/:N:/' $output | grep -v 'print-all-candidates' > $output.tmp
```

Comment:
Minor, but it seems like it'd be slightly preferable to replace "and 87 other" with "and nnn other " or the like so that someone reading the .bad without reading the .prediff wouldn't think there were no other candidates.

Reply:
Done